### ISSUE TITLE
Upgrade 5.3.3 to 5.4.6

### ISSUE THREAD

**Issue by CainDawson** [firefly-iii/firefly-iii#4061]
I'm using Firefly III version 5.3.3.

I wanted to upgrade from 5.3.3 to the current 5.4.6 via Softaculous. After the upgrade, I am told that I should go to the website and follow the last steps. Unfortunately I only get one white page.

I have the webspace package M at Contabo.de

This debug info was created before the upgrade.
Debug information generated at 2020-11-14 14:43:06 Europe / Berlin for Firefly III version ** 5.3.3 **.

| Scope | Version |
| --- | --- |
| Firefly III | 5.3.3 |
| Firefly III API | 1.3.0 |
| PHP | 7.4.9 |
| Host | Linux |

| System info | Value |
| --- | --- |
| Installation ID | d2dc9fb0-e7ff-4ab9-b7f9-0ef2ee2e3df2 |
| Using docker? | false |
| Telemetry | true |
| Layout | v1 |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | cgi-fcgi |
| Default language | de_DE |
| Default locale | equal |
| BCscale | 12 |
| DB drivers | mysql, sqlite |
| Current driver | mysql |
| Login provider | eloquent |
| Trusted proxies (.env) | |

| User info | Value |
| --- | --- |
| Session start | 2020-11-01 00:00:00 |
| Session end | 2020-11-30 23:59:59 |
| Session first | 2017-01-01 00:00:00 |
| User ID | 1 |
| User language | de_DE |
| User locale | de_DE |
| Attempt at "de_DE.utf8" | 'de_DE.utf8' |
| Attempt at "de_DE.UTF-8" | 'de_DE.UTF-8' |
| User agent | Mozilla / 5.0 (Windows NT 10.0; Win64; x64) AppleWebKit / 537.36 (KHTML, like Gecko) Chrome / 80.0.3987.163 Safari / 537.36 |

I only find the following entries under / storage / logs:

[2020-11-14 02:28:23] local.INFO: User visits homepage. {"path": "GET: http: //dateu.de/public", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 02:29:54] local.INFO: User visits homepage. {"path": "GET: http: //dateu.de/public", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 02:30:17] local.INFO: User visits homepage. {"path": "GET: http: //dateu.de/public", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 02:30:33] local.INFO: User visits homepage. {"path": "GET: https: //dateu.de/public", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 02:30:47] local.INFO: User visits admin index. {"path": "GET: https: //dateu.de/public/admin", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 02:30:59] local.INFO: User visits homepage. {"path": "GET: https: //dateu.de/public", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 02:33:59] local.INFO: User visits admin index. {"path": "GET: https: //dateu.de/public/admin", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 02:35:05] local.INFO: User is trying to login using "mail@dateu.de" {"path": "POST: https: //dateu.de/public/login", "IP": "REDACTED"}
[2020-11-14 02:35:05] local.INFO: User "mail@dateu.de" has been logged in. {"Path": "POST: https: //dateu.de/public/login", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 02:35:06] local.INFO: User visits homepage. {"path": "GET: https: //dateu.de/public", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 02:35:27] local.INFO: User visits admin index. {"path": "GET: https: //dateu.de/public/admin", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 02:35:37] local.INFO: User visits admin index. {"path": "GET: https: //dateu.de/public/admin", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 02:36:14] local.INFO: User visits admin index. {"path": "GET: https: //dateu.de/public/admin", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 02:36:21] local.INFO: User visits admin config index. {"path": "GET: https: //dateu.de/public/admin/configuration", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 02:36:28] local.INFO: User on index of link types in admin. {"path": "GET: https: //dateu.de/public/admin/links", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 02:36:54] local.INFO: User visits admin index. {"path": "GET: https: //dateu.de/public/admin", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 02:39:42] local.INFO: User visits admin index. {"path": "GET: https: //dateu.de/public/admin", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 02:49:58] local.INFO: User visits homepage. {"path": "GET: http: //dateu.de/public", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 08:04:54] local.INFO: User visits homepage. {"path": "GET: http: //dateu.de/public", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 08:05:15] local.INFO: User visits homepage. {"path": "GET: http: //dateu.de/public", "IP": "REDACTED", "user": "mail@dateu.de"}
[2020-11-14 08:05:33] local.INFO: Creating new account. {"path": "GET: http: //dateu.de/public/accounts/create/liabilities", "IP": "REDACTED", "user": "mail@dateu.de"}

I have this under / storage / logs / after the upgrade, because I only get a white page:

[2020-11-14 14:51:55] local.WARNING: The current configured Firefly III version (5.3.3) is older than the required version (5.4.6). Redirect to migrate routine.

What can I do now?

**Comment by CainDawson** [comment]
Even if I reinstall version 5.4.6 via Softacolous, I only get a white page with the following /storage/logs/

[2020-11-14T00:43:15.270645+01:00] local.INFO: AUDIT: Show login form (1.1). (ZENSIERT -> GET:http://test.dateu.de/public/login) [] []

**Comment by JC5** [comment]
I have no idea, I don't do Softaculous support. Without errors I can't point you in the right direction.

**Comment by CainDawson** [comment]
Is there any good tutorial with Docker other than the one in Firefly's docs?

---

Gibt es eine Gute Anleitung mit Docker außer die in den Docs von Firefly?

**Comment by da4id** [comment]
Hi i upgraded from 5.3.2 to 5.4.6 and i worked fine. I use docker too.

for the upgrade i used the following 3 commands
`docker stop <ID>
docker rm <ID>
docker pull jc5x/firefly-iii:latest`

replace <ID> with the current container ID. You can find out your ID with
`docker container ls`

If you have a Backup of your Database you could stop and remove the current firefly container. Restore the Database Backup and try it again. I had similar problems in the past and this procdure helped solving it.

**Comment by CainDawson** [comment]
Hello everybody,

I only have one web hosting package. Shell commands cannot be executed there. I have now done the following. Since I still have a VPS, I tried back and forth. Then I came across Cloudron. Perfect solution. Firefly runs fine with Cloudron on my VPS. :-) I have already saved all of the data that was important to me.

I am now very happy that it works and that without dropouts.

Lg. CainDawson

--

Hallo alle miteinander,

Ich habe nur ein Webhosting Paket. Da lassen sich Shell Befehle nicht ausführen. Folgendes habe ich nun gemacht. Da ich noch einen VPS habe, habe ich hin und her Probiert. Dann kam ich auf Cloudron. Perfekte Lösung. Firefly Läuft problemlos mit Cloudron auf meinen VPS. :-) Daten habe ich auch schon alle hinterlegt was wichtig war für mich.

Bin jetzt wieder sehr glücklich das es funktioniert und das ohne aussetzer.

Lg. CainDawson
